### PR TITLE
chore: bump gcloud version to 398.0.0

### PR DESCRIPTION
**Pull request by jianglai**
This should fix https://github.com/spinnaker/spinnaker/issues/6699 where the latest Spinnaker version fails to deploy to App Engine using an `appengine-web.xml` file containing a new tag (`<app-engine-apis>true</app-engine-apis>`) that was [introduced recently](https://cloud.google.com/appengine/docs/standard/java-gen2/services/access).

### CODE REVIEW

**Comment by jianglai** [comment]
Ping. Can we have this merged and released?

**Comment by jianglai** [comment]
Hi, can we get a review? This should be a trivial change.

File: Dockerfile.slim [hunk 1]
```diff
@@ -6,7 +6,7 @@ ENV KUBECTL_RELEASE=1.20.6
 ENV AWS_CLI_VERSION=1.18.152
 ENV AWS_CLI_S3_CMD=2.0.2
 ENV AWS_AIM_AUTHENTICATOR_VERSION=0.5.5
-ENV GOOGLE_CLOUD_SDK_VERSION=374.0.0
+ENV GOOGLE_CLOUD_SDK_VERSION=396.0.0
```
Comment (dbyron-sf) [inline Dockerfile.slim:9]: Adjust [Dockerfile.ubuntu](https://github.com/spinnaker/clouddriver/blob/341706911c97af883d57e949fd20598472141dab/Dockerfile.ubuntu) as well please.

Comment (jianglai) [reply Dockerfile.slim]: Done. Also bumped the version again to 398.0.0.

PTAL.

**Review by dbyron-sf — APPROVED** [review]
LGTM.

**Comment by jianglai** [comment]
Thank you. Do you know when can I expect a new release to be cut that includes this patch?

**Comment by dbyron-sf** [comment]
> Thank you. Do you know when can I expect a new release to be cut that includes this patch?

Not 100% sure, but with any luck 1.29.0 will appear in the next week or two.

**Pull request merged**